src/FakeItEasy/Core/DefaultFakeManagerAccessor.cs

```diff
@@ -26,7 +26,7 @@ public FakeManager GetFakeManager(object proxy)
 
             if (result == null)
             {
-                throw new ArgumentException("The specified object is not recognized as a fake object.");
+                throw new ArgumentException(string.Format(ExceptionMessages.NotRecognizedAsFake, proxy.ToString(), proxy.GetType().FullNameCSharp()));

```

Comment:
I appreciate the change to American spelling.
However, we'd also talked about [adding an `A` to the name](https://github.com/FakeItEasy/FakeItEasy/pull/1007#discussion_r105146897). Do you still agree?